modules/tinymce/src/core/test/ts/browser/content/EditorContentTest.ts

```diff
@@ -269,6 +269,25 @@ describe('browser.tinymce.core.content.EditorContentTest', () => {
           const tree = editor.getContent({ format: 'tree' });
           assert.equal(toHtml(tree), '<p>replaced</p>', 'Should be replaced html');
         });
+
+        const INITIAL_CONTENT = '<p>initial</p>';
+        const NEW_CONTENT = '<p>new content</p>';
+        const MANIPULATED_CONTENT = '<p>manipulated</p>';
```

Comment:
Camelcase these seems that we settled on using camel case.

Reply:
Sorry, forgot to change. Get a bit used to such constants wording. Updated [here](https://github.com/tinymce/tinymce/pull/8215/commits/bd4f3b04de484d0738dc2fc71d59cf2b2f89a8cd).